File: cpp/ACMP907.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main() {
    int w, h, r;
    cin >> w >> h >> r;

    if (min(w, h) / 2 >= r) {
        cout << "YES" << endl;
    } else cout << "NO" << endl;

    return 0;
}

File: src/ACMP907.java
import java.io.PrintWriter;
import java.util.Scanner;

public class ACMP907 {
    // your solution here
    private void solve(Scanner in, PrintWriter out) {
        int w = in.nextInt(),
            h = in.nextInt(),
            r = in.nextInt();

        if (Math.min(w, h) / 2 >= r) out.println("YES");
        else out.println("NO");
    }

    void run() {
        try (
            Scanner in = new Scanner(System.in);
            PrintWriter out = new PrintWriter(System.out)
        ) {
            solve(in, out);
        }
    }

    public static void main(String[] args) {
        new ACMP907().run();
    }
}
